std/concurrency.d

```diff
@@ -2473,6 +2474,23 @@ private
 
 version( unittest )
 {
+    void changeScheduler(Scheduler s)
+    {
+        import core.memory : GC;
+        import core.thread : Thread;
+
+        bool dosleep;
```

Comment:
It would be nice to have this explicitly initialised to make the intent clearer.